deps/build.jl

```diff
@@ -1,31 +1,63 @@
-using BinDeps, Compat
-@BinDeps.setup
+using Compat
 
-@osx_only Libdl.dlopen("libstdc++",Libdl.RTLD_GLOBAL)
+import Compat: String
 
-cpxvers = ["124","125","1251","1260","1261","1262","1263"]
+depsfile = joinpath(dirname(@__FILE__),"deps.jl")
+if isfile(depsfile)
+    rm(depsfile)
```

Comment:
this is going to cause unnecessary recompilation, would be better to only write to the file if the content changes